scripts/umi_binning.sh

```diff
@@ -378,6 +378,11 @@ $BWA aln $BINNING_DIR/reads_tf_umi2.fa $BINNING_DIR/umi_ref_b2.fa \
 $BWA samse -n 10000000 $BINNING_DIR/reads_tf_umi2.fa $BINNING_DIR/umi2_map.sai\
   $BINNING_DIR/umi_ref_b2.fa | $SAMTOOLS view -F 20 - > $BINNING_DIR/umi2_map.sam
 
+#CS: The gawk command below takes our two umi_map.sam files and extract reads mapped to umi bins. Following the read extraction, reads are filtered 
```

Comment:
I don't understand what this function does from this summary.  You go into the technical details without really getting to the heart of what this function is trying to do.  A simple 1-2 sentence summary is what's needed here.

Reply:
Sorry I changed it in the latest edit. The gawk command uses the sam files to put reads into groups and does QC on the groups. Groups passing QC are then passed on for consensus generation of highly accurate reads.